dashboards/create-metrics.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import json
+import logging
+from google.cloud import logging as gcloud_logging
+
+
+def main(service_account_path, pre_defined_metrics_json='metrics_template.json', google_project=None):
+    logging.basicConfig(level=logging.INFO)
+
+    # Instantiate a client from the service account JSON key path
+    logging_client = gcloud_logging.Client.from_service_account_json(service_account_path)
+
+    # Load pre-defined metrics JSON
+    with open(pre_defined_metrics_json, 'r') as f:
+        METRICS = json.load(f)
+
+    # List the current metrics
+    list_metrics(logging_client)
+
+    # Create the necessary metrics from the pre-defined list
+    if google_project:
```

Comment:
Can you clarify why `google_project` is included as an option here? If it is not specified, the script will set a default anyways so I'm just wondering why it is needed

Reply:
Good point! This is more like a dual fail-safe here. If the user passes in a `google_project` here which is inconsistent with the project associated with the service account key, this will cry. We could remove it later if this is not that helpful as I expect!